apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -97,58 +97,26 @@ public abstract partial class BigtableClient
-
-            var endpoint = clientCreationSettings?.ServiceEndpoint ?? BigtableServiceApiClient.DefaultEndpoint;
-            var clients = new BigtableServiceApiClient[clientCount];
-            var shutdowns = new Func<Task>[clientCount];
-            // Fill clients[] with BigtableServiceApiClient instances, each with specific channel
-            for (int i = 0; i < clientCount; i++)
-            {
-                var channel = new Channel(endpoint.Host, endpoint.Port, channelCredentials);
-                clients[i] = BigtableServiceApiClient.Create(channel, clientCreationSettings?.BigtableServiceApiSettings);
-                shutdowns[i] = channel.ShutdownAsync;
-            }
-            Func<Task> shutdown = () => Task.WhenAll(shutdowns.Select(x => x()));
-
-            return new BigtableClientImpl(appProfileId, clients) { UnderlyingClientSettings = settings };
-        }
+        public static BigtableClient Create(ClientCreationSettings clientCreationSettings = null, string appProfileId = null) =>
+            Task.Run(() => CreateAsync(clientCreationSettings, appProfileId)).ResultWithUnwrappedExceptions();
 
         /// <summary>
-        /// Creates a <see cref="BigtableClient"/> which uses the specified channel for remote operations.
+        /// Synchronously creates a <see cref="BigtableClient"/>, applying defaults for all unspecified settings,
+        /// and creating a channel connecting to the given endpoint with application default credentials where
+        /// necessary.
         /// </summary>
-        /// <param name="channel">The <see cref="Channel"/> for remote operations. Must not be null.</param>
-        /// <param name="settings">Optional <see cref="BigtableServiceApiSettings"/>.</param>
+        /// <param name="clients">The <see cref="BigtableServiceApiClient"/>s to use in a <see cref="BigtableClient"/>.</param>
         /// <param name="appProfileId">
         /// This is a private alpha release of Cloud Bigtable replication. This feature
         /// is not currently available to most Cloud Bigtable customers. This feature
         /// might be changed in backward-incompatible ways and is not recommended for
         /// production use. It is not subject to any SLA or deprecation policy.
-        ///
         /// This value specifies routing for replication. If not specified, the
         /// "default" application profile will be used.
         /// </param>
         /// <returns>The created <see cref="BigtableClient"/>.</returns>
-        public static BigtableClient Create(Channel channel, BigtableServiceApiSettings settings = null, string appProfileId = null)
-        {
-            var client = BigtableServiceApiClient.Create(channel, settings);
-            return new BigtableClientImpl(appProfileId, client) { UnderlyingClientSettings = settings };
-        }
+        public static BigtableClient Create(BigtableServiceApiClient[] clients, string appProfileId = null) => 
```

Comment:
This should take `IEnumerable<BigtableServiceApiClient>` as the first parameter and use `ToArray()` below after validating that the parameter is non-null:
```csharp
new BigtableClientImpl(GaxPreconditions.CheckNotNull(clients, nameof(clients)).ToArray(), appProfileId);
```